File: cpp-test-files/13.cpp
//define
#include <iostream>
#include <vector>

#define ZIPCODE 1234

int main() {
    std::vector<int> postalCodes = {3333, 4444, 5555, 1234, 9999};

    for (int code : postalCodes) {
        if (code == ZIPCODE) {
            std::cout << "Found ZIP code: " << ZIPCODE << "!\n";
        }
    }

    return 0;
}

File: java-files/13.java
public class IteratorExample {
    public static void main(String[] args) {
        List<Integer> postalCodes = new ArrayList<>(Arrays.asList(3333, 4444, 5555, 1234, 9999));
        for (int code : postalCodes) {
            if (code == Integer.parseInt("Z")) {
                System.out.println("Found ZIP code: " + code + "!");
            }
        }
    }
}